CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -981,7 +981,16 @@ explosive ordinance. When active, this system destroys any incoming grenades,
 missiles, rockets or thrown weapons that it has line of sight to from the top of 
 its 3m mast within a 6m radius. Any incoming munitions are destroyed before they 
 can detonate. Cannot stop incoming bullets, lasers, or plasma. Will stop some 
-hostile Nanite attacks, see GM for specifics.
+hostile Nanite attacks, see GM for specifics. 
+
+    The sensors that detect incoming explosive ordinance are rather sensitive
+and can easily be disrupted with a hacking DC check of 45 as long as the hacker
+has an access point within 20m. If a hacker surpasses a DC of 120, the ASPRO-S
+can be pwned and all of the defusing charges can be used to make a single 
```

Comment:
Questionable use of defuse, propose interception charges or just "all of its ammunition"